docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3785,7 +3791,8 @@ exact `action` specification in one `ActionProfileActionSet` message. However,
 the P4Runtime client is encouraged not to do so, as the same can be achieved by
 using the `weight` field. Note that to preserve read-write symmetry, the server
 must not coalesce multiple `ActionProfileAction` messages with the same `action`
-specification into one.
+specification into one. Additionally, each `action` specification would count as
+a separate member for the purposes of &eg; `max_num_members`.
```

Comment:
What does `&eg;` do, is this used here on purpose?

Reply:
Yes. It is `e.g.`, but apparently the markdown version?